### PR TITLE
added routes from products in home page to single product page , and …

### PR DESCRIPTION
**Pull request by saharSaleh22**
…deleted static data files and used data from mongoDB instead

### CODE REVIEW

File: src/Router.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ const Router = () => {
           <Route index element={<Login />} />
           <Route path="/HomePage" element={<HomePage />} />
           <Route path="/singleProduct" element={<SingleProduct />} />
+          <Route path="/HomePage/:id" element={<SingleProduct />} />
```
Comment (soraabunasser1993) [inline src/Router.js:69]: I think the path should be path="/products/:id"

File: src/components/Login/AuthForm.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    console.log("inside signup ");
+  }
+  return (
+    <form
+      action=""
+      className={props.type === "signin" ? "sign-in-form" : "sign-up-form"}
+      onSubmit={(e) => {
+        e.preventDefault(); // Prevent form submission from refreshing the page
+        // Call your submit logic here
+        if (props.type === "signin") {
```
Comment (soraabunasser1993) [inline src/components/Login/AuthForm.js:38]: please replace this with a ternary operator 
`{props.type === "'signin" ? logIn() : signUp()}`

File: src/components/Login/SubmitButton.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
 import React from "react";
 function Submit(props) {
-  console.log(props.class);
   return (
-    <input
+    <button
```
Comment (soraabunasser1993) [inline src/components/Login/SubmitButton.js:4]: why you are not using mui button?

File: src/components/ProductsHomePage/ProductButton.js [hunk 4]
```diff
@@ -1,7 +1,14 @@
 import React from "react";
 import { ButtonForProduct } from "../../StyledComponents";
+import { Link } from "react-router-dom";
 
 function ProductButton(props) {
-  return <ButtonForProduct size="small">{props.value}</ButtonForProduct>;
+  return (
+    <Link to={`/HomePage/${props.productId}`}>
```
Comment (soraabunasser1993) [inline src/components/ProductsHomePage/ProductButton.js:7]: This should be update once path updated

File: src/Router.js (hunk 1 above)
Comment (saharSaleh22) [reply src/Router.js:69]: yes yes , i will do that soon .

File: src/components/Login/AuthForm.js (hunk 2 above)
Comment (saharSaleh22) [reply src/components/Login/AuthForm.js:38]: done

File: src/components/ProductsHomePage/ProductButton.js (hunk 4 above)
Comment (saharSaleh22) [reply src/components/ProductsHomePage/ProductButton.js:7]: ok.

Comment (saharSaleh22) [reply src/components/ProductsHomePage/ProductButton.js:7]: done.

File: src/Router.js (hunk 1 above)
Comment (saharSaleh22) [reply src/Router.js:69]: done.

File: src/components/Login/SubmitButton.js (hunk 3 above)
Comment (saharSaleh22) [reply src/components/Login/SubmitButton.js:4]: when i am finishing the project , in final touches i will convert it to mui button : ).

**Pull request merged**